package.json

```diff
@@ -23,6 +23,7 @@
     "test": "jest --passWithNoTests",
     "test:coverage": "curl -sSL 'https://raw.githubusercontent.com/RedHatInsights/insights-interact-tools/refs/heads/main/scripts/coverage.sh' | bash",
     "test:ct": "BABEL_ENV=componentTest cypress run --browser chrome --component",
+    "test:ctspec": "BABEL_ENV=componentTest cypress run --browser chrome --component --spec",
```

Comment:
Can't you just run "npx cypress open", open the browser, and run the test forthe  1 file you want? You don't need to run it headless like this